src/runtime-rs/crates/hypervisor/src/dragonball/mod.rs

```diff
@@ -75,9 +75,18 @@ impl Hypervisor for Dragonball {
         id: &str,
         netns: Option<String>,
         _annotations: &HashMap<String, String>,
+        // Dragonball is a built-in VMM that runs as a thread inside containerd-shim-kata-v2.
+        // Because it is not a standalone process, Dragonball cannot independently set per-VM SELinux exec labels;
+        // any provided selinux_label will be ignored.
```

Comment:
Move these lines to the beginning of the function, and start with `///`. You should wrap the line to prevent it from becoming too long (75 charactors per line). For example:

```
/// Dragonball is a built-in VMM that runs as a thread inside the
/// runtime. Because it is not a standalone process, Dragonball cannot
/// independently set per-VM SELinux exec labels; any provided
/// selinux_label will be ignored.
#[instrument]
async fn prepare_vm(
```